### PR TITLE
Andrzej

### PR DESCRIPTION
**Pull request by andpodob**
I think I handled the mess that i've accidentally produced.

### CODE REVIEW

**Comment by yongli-abc** [comment]
I saw you have several merge commits (e.g. 8e3a8c7, a00fbe4) that merge 'master' into 'andrzej'. This should not be needed as master branch is meant to contain the most stable version, and you should not be in a situation where you need to actively backport changes into your dev branch, especially this repo is currently only owned by yourself.

Usually I would suggest having new branch for each new feature rather than keeping a long-living dev branch. Whenever you start a new feature, branch off the latest master HEAD so you always start with the latest master version.

When you're collaborating with others on a repo, it's possible that the master branch will have conflict with your ongoing dev branch. My general practice is to wait until the PR merge time to resolve conflicts. If you really want to sync your current dev branch with the latest master state, I would strongly suggest rebasing the branch onto master HEAD rather than merge back from master.

The general goal is to have short-lived feature branches contribute to the growth of master branch (and avoid the other way around), in order to achieve a linear and readable commit history :)

File: portfolio/src/main/webapp/script.js
Comment (yongli-abc) [inline portfolio/src/main/webapp/script.js]: Default to use `const`, until you actually need to mutate the variable, then change to `var`.

Comment (yongli-abc) [reply portfolio/src/main/webapp/script.js]: Same for all `var`s below.

File: portfolio/src/main/webapp/script.js
Comment (yongli-abc) [inline portfolio/src/main/webapp/script.js]: remove extra line

**Review by yongli-abc — CHANGES_REQUESTED** [review]

**Comment by andpodob** [comment]
Ok, thanks thanks for your worthy advices, I've done these small changes. Starting from yesterday I will make every feature in other branch. Thanks.

**Review by yongli-abc — APPROVED** [review]

**Pull request merged**
